pyproject.toml

```diff
@@ -4,7 +4,21 @@ requires = [
     "packaging",
     "wheel",
     "cython>=0.29.20",
-    "numpy>=1.16.6,<1.20",
+    # See https://numpy.org/doc/stable/user/depending_on_numpy.html for
+    # the recommended way to build against numpy's C API:
+    "oldest-supported-numpy",
     "scipy>=1.0",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.cibuildwheel]
+manylinux-x86_64-image = "manylinux2014"
+manylinux-i686-image = "manylinux2014"
+# Change in future version to "build"
+build-frontend = "pip"
+
+[tool.cibuildwheel.override]
```

Comment:
What's the difference between these two sections?

Reply:
See previous reply. The `.override` just specifies `manylinux2010` for cases where it works and where we've used it to build previous releases (i.e. 4.6.x and not Python 3.10).